opendatahub-io/odh-dashboard
# Add details view for pipeline jobs

closes: https://github.com/opendatahub-io/odh-dashboard/issues/1243

## Description


Scheduled runs table now allows you to link to a scheduled runs details page. This page is the same as the pipeline runs details page except that the graph cannot be selected, the bottom drawer omits the yaml results, and the drawer details tab has less info because it has not run yet (duration, end time, ...)
![image](https://github.com/opendatahub-io/odh-dashboard/assets/12587674/c1cca501-ad5e-40a4-bae9-4097da8ec0df)
![image](https://github.com/opendatahub-io/odh-dashboard/assets/12587674/e48386d7-47b3-42e7-96d2-108cf2d6238f)

<img width="189" alt="Screenshot 2023-09-20 at 4 01 31 PM" src="https://github.com/opendatahub-io/odh-dashboard/assets/12587674/2e06c1e1-8e40-442e-9ea1-3df393579768">

## How Has This Been Tested?



1. create a pipeline server and add a pipeline scheduled run
2. verify you can navigate to the details page from the scheduled runs table by selecting the run name.
3. verify that the details tab is correct and there is no yaml shown
5. verify you can delete and duplicate the run
6. verify the graph is selectable for already run runs and pipeline view.

## Test Impact


I added tests to make sure that elements that are in a traditional ran run are not in the scheduled run detail page (finished, duration, graph clicking, run output)

## Request review criteria:



Self checklist (all need to be checked):

If you have UI changes: 


After the PR is posted & before it merges:
